README.md

```diff
@@ -31,33 +46,41 @@ getting started, or view the full [Conduit roadmap][roadmap].
 
 1. Install the Conduit CLI with `curl https://run.conduit.io/install | sh `.
 
-2. Add `$HOME/.conduit/bin` to your `PATH`.
+1. Add `$HOME/.conduit/bin` to your `PATH`.
 
-3. Install Conduit into your Kubernetes cluster with:
-  `conduit install | kubectl apply -f -`.
+1. Install Conduit into your Kubernetes cluster with `conduit install | kubectl apply -f -`.
 
-4. Verify that the installation succeeded with `conduit check`.
+1. Verify that the installation succeeded with `conduit check`.
 
-5. Explore the Conduit controller with `conduit dashboard`.
+1. Explore the Conduit controller with `conduit dashboard`.
 
-6. Optionally, install a [demo application][conduit-demo] to run with Conduit.
+1. Optionally, install a [demo application][conduit-demo] to run with Conduit.
 
-7. Add [your own service][conduit-inject] to the Conduit mesh!
+1. Add [your own service][conduit-inject] to the Conduit mesh!
 
 ## Working in this repo ##
 
 [`BUILD.md`](BUILD.md) includes general information on how to work in this repo.
 
+We :heart: pull requests! See [CONTRIBUTING.md](CONTRIBUTING.md) for info on
```

Comment:
Can you put `CONTRIBUTING.md` in backticks here, to match how BUILD.md is linked on the previous line?

```
We :heart: pull requests! See [CONTRIBUTING.md](`CONTRIBUTING.md`) for info on
```